packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardPreview.styles.ts

```diff
@@ -0,0 +1,83 @@
+import { getGlobalClassNames, FontSizes } from '../../Styling';
+import { IDocumentCardPreviewStyleProps, IDocumentCardPreviewStyles } from './DocumentCardPreview.types';
+
+export const GlobalClassNames = {
+  root: 'ms-DocumentCardPreview',
+  icon: 'ms-DocumentCardPreview-icon',
+  iconContainer: 'ms-DocumentCardPreview-iconContainer',
+  previewFileTypeIcon: 'ms-DocumentCardPreview-icon'
```

Comment:
Does this suppose to have the same global className as on line 6? Maybe leave just `GlobalClassNames.icon` and use it in both places?

Reply:
Thanks for catching that, it should have a different global class name.